packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -326,15 +316,15 @@ class RenderStack extends RenderBox
     AlignmentGeometry alignment = AlignmentDirectional.topStart,
     TextDirection textDirection,
     StackFit fit = StackFit.loose,
-    Overflow overflow = Overflow.clip,
+    Clip clipBehavior = Clip.none,
```

Comment:
why not clip with the hard edge by default? This only kicks in if there's overflow, so it's not like it makes much difference in most cases.

Reply:
Changed it to hard edge by default. I previously was following https://github.com/flutter/flutter/issues/21830#issuecomment-421141732.